<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1278</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="buttonTest1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Test1 : Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonTest3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Test3 : Accelerometer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonTest2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Test2 : Info &amp; LED</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonTest4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Test4 : Gyro</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonTest5">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Test5 : Sensor Fusion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonTest6">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>10</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Test6 : Accel + Gyro (Dont used)</string>
    </property>
   </widget>
   <widget class="QWidget" name="accelXWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>681</width>
      <height>591</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="accelYWidget" native="true">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>421</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="accelZWidget" native="true">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>90</y>
      <width>421</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="rotateXWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>421</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="rotateYWidget" native="true">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>370</y>
      <width>421</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="rotateZWidget" native="true">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>370</y>
      <width>421</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1278</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
